services/eventsapi/main.py

```diff
@@ -69,4 +74,11 @@ async def create_event(event: Event):
                 Body=event.json()
             )
     else:
-        print(f"Received event: {event.json()}")
\ No newline at end of file
+        print(f"Received event: {event.json()}")
+
+    if KAFKA_BROKERS:
+        producer = KafkaProducer( bootstrap_servers=KAFKA_SERVER, api_version=(2, 0, 2))
+        producer.send(KAFKA_TOPIC, event.json().encode('utf-8'))
```

Comment:
This doesn't really cause problems right now, but when we have schemas associated with topics it's likely going to mean we'll have a event type per topic. Just to emulate that model here, is it possible to use the event's name as the topic? The only ones that might be problematic are the `LessonContentPageViewedEvent` and `UserGradedEvent` events since they use the full Moodle namespaced string values (though we can tweak that if needed).

Reply:
All of this is admittedly just from kicking the can earlier. I'd suggest we go ahead and rename the events in the spike code so we don't have two different formats (we can discuss later what "real" naming conventions we want for the actual implementation). Maybe?:

* `\\mod_lesson\\event\\content_page_viewed` becomes `moodle_lesson_content_page_viewed` (change [here](https://github.com/openstax/raise-moodle-spikes/blob/7c5687b17c60972fb3123db9af1469c86d924187/services/eventsapi/main.py#L29) and [here](https://github.com/openstax/raise-moodle-spikes/blob/7c5687b17c60972fb3123db9af1469c86d924187/plugins/events/classes/event_processor.php#L39))
* `\\core\\event\\user_graded` becomes `moodle_user_graded`  (change [here](https://github.com/openstax/raise-moodle-spikes/blob/7c5687b17c60972fb3123db9af1469c86d924187/services/eventsapi/main.py#L38) and [here](https://github.com/openstax/raise-moodle-spikes/blob/7c5687b17c60972fb3123db9af1469c86d924187/plugins/events/classes/event_processor.php#L58))